### PR TITLE
feat(nav-bar-content): added new reflow version of assessment-view 

### PR DESCRIPTION
**Pull request by JGibson2019**
with logic to show the gettingstarted view when selected subview is not a requirement. Also added a test flie and appropriate snapshot

#### Pull request checklist

- [ ] Addresses an existing issue: #0000
- [X] Ran `yarn fastpass`
- [X] Added/updated relevant unit test(s) (and ran `yarn test`)
- [X] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [X] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). See `CONTRIBUTING.md`.
- [N/A] (UI changes only) Added screenshots/GIFs to description above
- [N/A] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/DetailsView/components/reflow-assessment-view.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    AssessmentData,
+    AssessmentNavState,
+    PersistedTabInfo,
+} from '../../common/types/store-data/assessment-result-data';
+import { GettingStartedView } from './getting-started-view';
+import { TargetChangeDialog, TargetChangeDialogDeps } from './target-change-dialog';
+
+export type WithAssessmentTestResult = { assessmentTestResult: AssessmentTestResult };
+
+export type ReflowAssessmentViewDeps = ContentLinkDeps & TargetChangeDialogDeps;
```
Comment (waabid) [inline src/DetailsView/components/reflow-assessment-view.tsx:18]: Is the contentlinkdeps being used?

File: src/DetailsView/components/reflow-assessment-view.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import { Tab } from '../../common/itab';
+import {
+    AssessmentData,
+    AssessmentNavState,
+    PersistedTabInfo,
+} from '../../common/types/store-data/assessment-result-data';
+import { GettingStartedView } from './getting-started-view';
+import { TargetChangeDialog, TargetChangeDialogDeps } from './target-change-dialog';
+
+export type WithAssessmentTestResult = { assessmentTestResult: AssessmentTestResult };
```
Comment (waabid) [inline src/DetailsView/components/reflow-assessment-view.tsx:16]: where is this being used?

Comment (JGibson2019) [reply src/DetailsView/components/reflow-assessment-view.tsx:16]: AssessmentTestResult is used to pass the getting started information for each assessment

File: src/DetailsView/components/reflow-assessment-view.tsx (hunk 1 above)
Comment (JGibson2019) [reply src/DetailsView/components/reflow-assessment-view.tsx:18]: No it's not, so going to remove it! Moving forward, should I continue using `ReflowAssessmentViewDeps`? 

so option 1:  `export type ReflowAssessmentViewDeps = TargetChangeDialogDeps;'  and keep props as is
option 2: delete `ReflowAssessmentViewDeps` and updated the props to say `deps: TargetChangeDialogDeps`

File: src/DetailsView/components/reflow-assessment-view.tsx (hunk 2 above)
Comment (JGibson2019) [reply src/DetailsView/components/reflow-assessment-view.tsx:16]: removing it!

**Pull request merged**